pkgs/applications/misc/furtherance/default.nix

```diff
@@ -0,0 +1,55 @@
+{ lib, stdenv, fetchFromGitHub, rustPlatform, appstream-glib, desktop-file-utils
+, glib, libadwaita, meson, ninja, pkg-config, python3, wrapGAppsHook4, dbus
+, gtk4, sqlite }:
+
+stdenv.mkDerivation rec {
+  pname = "furtherance";
+  version = "1.5.1";
+
+  src = fetchFromGitHub {
+    owner = "lakoliu";
+    repo = "Furtherance";
+    rev = "v${version}";
+    sha256 = "oIzwFPJUY+OwT2B6n7NPg8bMxEd4D0MKPe3GQxrXmEU=";
+  };
+
+  cargoDeps = rustPlatform.fetchCargoTarball {
+    inherit src;
+    name = "${pname}-${version}";
+    sha256 = "VlV6EkMKMGoUELUMgEp/OXdTJTifBqsoqrf/eZTyl1k=";
```

Comment:
Looks like this is incorrect, according to the build logs. (Any clue what happened on your end to cause this, out of curiosity?)

Reply:
That is very odd, other than changing the capitalization of pname, I can't think of any significant changes, but this is coming from cargo, I adjusted the hash and now it builds again. 

I did the rebase again after the fixup, did i do it correctly?

Reply:
> other than changing the capitalization of pname

I tested this, and that was the issue.

> I did the rebase again after the fixup, did i do it correctly?

Yup!